models/VolunteeringEvent.js

```diff
@@ -3,11 +3,11 @@ const mongoose = require('mongoose');
 const volunteeringEventSchema = new mongoose.Schema({
   request: {
     type: mongoose.Schema.Types.ObjectId,
-    ref: 'VolunteeringRequest',
+    ref: 'VolunteeringOfferRequestBase',
```

Comment:
I don't like it, but I guess there's no better way. Can you please check again if this change is necessary?

Reply:
We can change the collection name to something like "VolunteeringOffersRequets" (instead VolunteeringOfferRequestBase) but to split this into two collections it's not possible.

Reply:
I prefer it as is then.